src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.ladder.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Line {
+    private static final int MIN_LINE_SIZE = 1;
+
+    private final List<Point> line;
+
+    private Line(List<Point> line) {
+        validateLine(line);
+        this.line = line;
+    }
+
+    public static Line from(List<Point> line) {
+        return new Line(line);
+    }
+
+    private static void validateLine(List<Point> line) {
+        if (Objects.isNull(line) || line.size() < MIN_LINE_SIZE) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public List<Point> getPoints() {
+        return line;
```

Comment:
컬렉션을 그대로 반환하면 변조를 할 수 있어 일급 컬렉션의 의미가 없어지는 것 같아요.
내부 요소를 수정하지 못하는 컬렉션을 반환하면 어떨까요?
```suggestion
        return Collections.unmodifiableList(line);
```